recipe/patches/0001-Redis-deps-now-build-but-do-not-link.patch

```diff
@@ -199,5 +199,5 @@ index d034387596..27c01310ed 100644
 +2.38.1.windows.1
 +
 -- 
-2.34.1
+2.38.1.windows.1
```

Comment:
Sidenote: If you generate the patches with `--no-signature`, we can avoid the churn in the footers (we should also add that to `ray-patching.md`...)